filecoin-proofs/src/api/mod.rs

```diff
@@ -111,12 +116,64 @@ pub fn get_unsealed_range<T: Into<PathBuf> + AsRef<Path>, Tree: 'static + Merkle
     // If the call to `extract_range` was successful, the `unsealed` vector must
     // have a length which equals `num_bytes_padded`. The byte at its 0-index
     // byte will be the the byte at index `offset_padded` in the sealed sector.
-    let written = write_unpadded(unsealed, &mut buf_writer, 0, num_bytes.into())
-        .with_context(|| format!("could not write to output_path={:?}", output_path.as_ref()))?;
+    let written = write_unpadded(unsealed, &mut unsealed_output, 0, num_bytes.into())
+        .with_context(|| "write_unpadded failed")?;
```

Comment:
Just FYI: you if you don't use any variable, but just a string, you can use `context()` instead of `with_context()` =>

```rust
.context("write_unpadded failed")?;
```

Reply:
done